### PR TITLE
feat(fluxcd): add completion for flux

### PR DESCRIPTION
**Pull request by serrovsky**
## Standards checklist:

- [x] The PR title is descriptive.
- [x] The PR doesn't replicate another PR which is already open.
- [x] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [x] The code is mine or it's from somewhere with an MIT-compatible license.
- [x] The code is efficient, to the best of my ability, and does not waste computer resources.
- [x] The code is stable and I have tested it myself, to the best of my abilities.
- [x] If the code introduces new aliases, I provide a valid use case for all plugin users down below.

## Changes:

- Add fluxcd cli as a plugin to enable autocompletion

## Other comments:
Done, after this comment on this issue >
https://github.com/ohmyzsh/ohmyzsh/pull/10197#issuecomment-1324766214
...

### CODE REVIEW

File: plugins/flux/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# FluxCD plugin
+
+This plugin adds completion for [Flux](https://helm.sh/), an open and extensible continuous delivery solution for Kubernetes. Powered by GitOps Toolkit.
```
Comment (carlosala) [inline plugins/flux/README.md:3]: Should we have https://fluxcd.io?

**Review by carlosala — CHANGES_REQUESTED** [review]

File: plugins/flux/README.md (hunk 1 above)
Comment (carlosala) [reply plugins/flux/README.md:3]: I'd also name the plugin `fluxcd`, just to be more semantic in terms of not confusing with other projects named flux, such as https://github.com/influxdata/flux

Comment (serrovsky) [reply plugins/flux/README.md:3]: Yes, make sense. I will change it right now

**Review by serrovsky — COMMENTED** [review]
Plugon renamed and link updated

**Comment by carlosala** [comment]
Thanks for your contribution!

**Pull request merged**